python/hyperonpy.cpp

```diff
@@ -275,6 +280,29 @@ PYBIND11_MODULE(hyperonpy, m) {
     m.def("vec_atom_push", [](CVecAtom vec, CAtom atom) { vec_atom_push(vec.ptr, atom_clone(atom.ptr)); }, "Push atom into vector");
     m.def("vec_atom_pop", [](CVecAtom vec) { return CAtom(vec_atom_pop(vec.ptr)); }, "Push atom into vector");
 
+    py::class_<CBindings>(m, "CBindings");
+    m.def("bindings_new", []() { return CBindings(bindings_new()); }, "New bindings");
+    m.def("bindings_free", [](CBindings bindings) { bindings_free(bindings.ptr);}, "Free bindings" );
+    m.def("bindings_clone", [](CBindings bindings) { return CBindings(bindings_clone(bindings.ptr)); }, "Deep copy if bindings");
+    m.def("bindings_merge", [](CBindings left, CBindings right) { return CBindings(bindings_merge(left.ptr, right.ptr));}, "Merges bindings");
+    m.def("bindings_eq", [](CBindings left, CBindings right){ return bindings_eq(left.ptr, right.ptr);}, "Compares bindings"  );
+    m.def("bindings_add_var_bindings",
+          [](CBindings bindings, char const* varName, CAtom atom) {
+              var_atom_t var_atom{.var = varName, .atom = atom.ptr};
```

Comment:
Here we should clone atom, because it is held by Python interpreter. If we pass pointer to the Rust and Rust move atom, Python interpreter will access to the memory which is already freed.
```suggestion
              var_atom_t var_atom{.var = varName, .atom = atom_clone(atom.ptr)};
```

Reply:
done